app/components/explore/side-bar.js

```diff
@@ -102,6 +112,13 @@ export default Component.extend({
     onDateChange() {
       this.send('selectDateFilter', 'custom_dates');
     },
+    clearFilterCategory() {
+      this.set('category', null);
+      this.set('sub_category', null);
```

Comment:
shouldn't we have separate option to clear `sub category` and `category`?

Reply:
@Anupam-dagar We are displaying both `SubCategory` and `Category` in one single filter option and I have provided an option for `All Category`, Now providing a way for `All Sub Category` and `All Category` would be tedious